checks.d/network.py

```diff
@@ -508,3 +554,71 @@ def _parse_solaris_netstat(self, netstat_output):
             metrics_by_interface[iface] = metrics
 
         return metrics_by_interface
+
+    def _linux_sockstat_get_conntrack_max(self):
```

Comment:
This function body repeats itself in two others. Let's condense the three into one, passing in the `proc_name`, instead.